lib/Watcher.js

```diff
@@ -126,6 +126,8 @@ export default class Watcher {
 
           ImportJS will fall back to polling instead. This is slower, and more
           resource-heavy. Plus, changes will not be immediately available.
+          \n
+          Reason: ${error.message}\n${error.stack}
```

Comment:
I think `error.stack` might include `error.message`, so this might be redundant, but NBD. (Although I don't think `error.stack` has been standardized yet, so maybe worth having both anyway I guess.)